dist/init.js

```diff
@@ -0,0 +1,65 @@
+import registerSettings, { FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME } from "./modules/settings.js";
+/**
+ * This is your TypeScript entry file for Foundry VTT.
+ * Register custom settings, sheets, and constants using the Foundry API.
+ * Change this heading to be more descriptive to your module, or remove it.
+ * Author: [your name]
+ * Content License: [copyright and-or license] If using an existing system
+ * 					you may want to put a (link to a) license or copyright
+ * 					notice here (e.g. the OGL).
+ * Software License: [your license] Put your desired license here, which
+ * 					 determines how others may use and modify your module
+ */
+// Import JavaScript modules
+// Import TypeScript modules
+import registerDerivedItemSheetClass from "./modules/ItemSheet.js";
+import registerContextMenuHook from "./modules/ContextMenu.js";
+import { checkSettingsInitialized, getGame } from "./modules/settings.js";
+import Identification from "./modules/Identification.js";
+import registerItemClassMethod from "./modules/Item.js";
+export let debugEnabled = 0;
+// 0 = none, warnings = 1, debug = 2, all = 3
+export let debug = (...args) => { if (debugEnabled > 1)
+    console.log(`DEBUG:${FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME} | `, ...args); };
+export let log = (...args) => console.log(`${FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME} | `, ...args);
+export let warn = (...args) => { if (debugEnabled > 0)
+    console.warn(`${FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME} | `, ...args); };
+export let error = (...args) => console.error(`${FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME} | `, ...args);
+export let timelog = (...args) => warn(`${FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME} | `, Date.now(), ...args);
+export let i18n = key => {
+    return getGame().i18n.localize(key);
+};
+export let i18nFormat = (key, data = {}) => {
+    return getGame().i18n.format(key, data);
+};
+export let setDebugLevel = (debugText) => {
+    debugEnabled = { "none": 0, "warn": 1, "debug": 2, "all": 3 }[debugText] || 0;
+    // 0 = none, warnings = 1, debug = 2, all = 3
+    if (debugEnabled >= 3)
+        CONFIG.debug.hooks = true;
+};
```

Comment:
- At first glance, this doesn't look quite correct - I believe `log` is missing a conditional?
- The log levels should have (unexported) constants
- I would suggest converting the arrow functions into regular function declarations. In this instance, using arrow functions doesn't really provide any benefit.
- Unless they are really simple (`if (disabled) return;`) `if` statements should use braces and have correct indentation (in other words, don't try to fit them on as few lines as possible).
- `CONFIG.debug.hooks` should not normally be set by modules.

Reply:
`
- At first glance, this doesn't look quite correct - I believe log is missing a conditional? **(Sorry could you be clearer?)**
- The log levels should have (unexported) constants **(yea nothing wrong with that)**
- I would suggest converting the arrow functions into regular function declarations. In this instance, using arrow functions doesn't really provide any benefit. **(not necessary but i can do that too)**
- Unless they are really simple (if (disabled) return;) if statements should use braces and have correct indentation (in other words, don't try to fit them on as few lines as possible). **(sure not a problem)**
- CONFIG.debug.hooks should not normally be set by modules.  **(it is not entirely true some modules implement it for example "midi-qol" (from which I took inspiration for the [functions](https://gitlab.com/tposney/midi-qol/-/blob/master/src/midi-qol.ts)), but I can remove it without problems)**

`

So just to be clear (taking just the log function as an example) I should replace this function:

`export let log = (...args) => console.log(`${FORIEN_UNIDENTIFIED_ITEMS_MODULE_NAME} | `, ...args);`

with this ?

`export let log = function(...args){ console.log(`forien-unidentified-items | `, ...args) };`